CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# Contributing
+
+This document provides some overall guidelines and suggestions for how to get started with contributing to the project. There are several features, enhancements, and fixes that we'd like to work on, so there are lots of opportunities to get started.
+
+## Table of Contents
+- [First steps](#first-steps)
+- [Contributing code](#contributing-code)
+- [Installation](#installation)
+- [Running Components](#running-the-docker-stack)
+
+### First steps
+
+1. [Set up your local dev environment](README.md#installation)
+2. Be sure to read our [code of conduct](CODE_OF_CONDUCT.md)
+3. Say hi on our Slack Channel!
+
+### Contributing code
```

Comment:
This is more along the lines of git basics rather than how to contribute. I don't think it belongs here. 

What I usually see in a projects contributing file are contributor expectations. 

For example an issue should not be created to ask a question, that is what the slack channel is for. Other expectations would be on how to claim issues: assign yourself the issue or leave a comment stating that your working on it. 

Expectation for PR's. Ideally each PR should be linked to an issue, and a keyword should be used to automatically close the issue when the PR is merged. 

A PR should be reviewed and approved by at least one member before being merged by the person who submitted the PR.

Reply:
Ok, I can add that stuff. Should I just move the PR command instructions back to the top-level readme?

Reply:
I was going to completely remove this and add it to the documentation site. Perhaps we can live this out of the readme now which will give me motivation to update the site.